packages/@pollyjs/persister-rest/src/index.js

```diff
@@ -4,10 +4,16 @@ import { buildUrl } from '@pollyjs/utils';
 import ajax from './ajax';
 
 export default class RestPersister extends Persister {
-  static get name() {
+  static get id() {
     return 'rest';
   }
 
+  static get name() {
```

Comment:
All of these name getter proxies need to sit on the persister/adapter classes instead of base classes because:

```js
class Foo {
        static get name() {
            return 'foo';
        }
    }

    class Bar extends Foo {}

    Bar.name; // "Bar" not "foo"
```